Cards/Resource/Card_DrawCards.tres

```diff
@@ -14,14 +13,9 @@ effect = SubResource("Resource_2ap55")
 value = 2
 target_type = 1
 
-[sub_resource type="Resource" id="Resource_j6isw"]
-script = ExtResource("3_06oqv")
-energy_cost = 1
```

Comment:
does that mean the DrawCard doesn't cost anything ? wouldn't it be better to have the energy cost of 0 to not try to get the energy info and getting null instead ?

Reply:
if the value isn't set then it will use the one set in cardbase. (when I renamed energy to energy info I had to reset the values for every card I may or may not have skipped it)

Reply:
I see, default is 0